cmd/otel-allocator/README.md

```diff
@@ -43,6 +43,20 @@ spec:
           processors: []
           exporters: [logging]
 ```
+## Set up filterStrategy
+`filterStrategy` determines how to filter targets before allocating them among multiple collectors. `relabel-config` is the currently supported option. With `filterStrategy`, targets will be consolidated based on Prometheus configuration `relabel_config`. By default, filtering is disabled.
+
+It's important to consider setting `filterStrategy` if you have a scrape job with a discovery role for example, `endpoints` which could produce multiple targets per endpoint. If these targets lead to the same metric after a relabeling step and they are assigned to different collectors, the outcome may be undesirable because duplicate time series are created for the metric. 
+
+For metrics collection pipeline where metrics are exported to Prometheus, you may see `out of order sample` error intermittenly if the time series produced have shifted time stamps from different collectors. 
```

Comment:
I'm not sure I've come across this issue nor if within this filter strategy doc comment is the right place to call this out.